skops/io/_utils.py

```diff
@@ -171,3 +171,21 @@ def get_state(value, save_context):
 
     res["__id__"] = __id__
     return res
+
+
+def get_type_name(
```

Comment:
Formatting of this function looks messed up, the whole `def` can fit on one line.